setup.py

```diff
@@ -78,7 +78,6 @@
     install_requires=[
         # if you add any additional libraries, please also
         # add them to `docs/requirements.txt`
-        'graphviz>=0.8',
```

Comment:
Should we add this to `EXTRAS_REQUIRE` so it is available to make docs, or is that unnecessary?

Reply:
Oh..I thought this was added; yes, I'll add it to `EXTRAS_REQUIRE`.